app/gui2/src/components/ExtendedMenu.vue

```diff
@@ -1,46 +1,49 @@
 <script setup lang="ts">
 import SvgIcon from '@/components/SvgIcon.vue'
+import { isMacLike } from '@/composables/events'
 import { ref } from 'vue'
 
 const isDropdownOpen = ref(false)
 
 const props = defineProps<{
   zoomLevel: number
 }>()
-const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: [] }>()
+const emit = defineEmits<{ zoomIn: []; zoomOut: []; fitToAllClicked: []; toggleCodeEditor: [] }>()
+
+const toggleCodeEditorShortcut = isMacLike ? 'Cmd + `' : 'Ctrl + `'
 </script>
 
 <template>
   <div class="ExtendedMenu">
-    <div class="moreIcon" @pointerdown="isDropdownOpen = !isDropdownOpen"></div>
+    <div class="moreIcon" @pointerdown="isDropdownOpen = !isDropdownOpen">…</div>
     <Transition name="dropdown">
       <div v-show="isDropdownOpen" class="ExtendedMenuPane">
         <div class="row">
-          <div class="zoomBar row">
-            <div class="label">Zoom</div>
-            <div class="zoomControl last">
-              <div
-                class="zoomButton minus"
-                title="Decrease zoom"
-                @pointerdown.stop="emit('zoomOut')"
-              />
-              <span
-                class="zoomScaleLabel"
-                v-text="props.zoomLevel ? props.zoomLevel.toFixed(0) + '%' : '?'"
-              ></span>
-              <div
-                class="zoomButton plus"
-                title="increase zoom"
-                @pointerdown.stop="emit('zoomIn')"
+          <div class="label">Zoom</div>
+          <div class="zoomControl">
+            <div
+              class="zoomButton minus"
+              title="Decrease zoom"
+              @pointerdown.stop="emit('zoomOut')"
+            />
+            <span
+              class="zoomScaleLabel"
+              v-text="props.zoomLevel ? props.zoomLevel.toFixed(0) + '%' : '?'"
+            ></span>
+            <div class="zoomButton plus" title="increase zoom" @pointerdown.stop="emit('zoomIn')" />
+            <div class="divider"></div>
+            <div class="showAllIconHighlight">
+              <SvgIcon
+                name="show_all"
+                class="showAllIcon"
+                @pointerdown.stop="emit('fitToAllClicked')"
               />
             </div>
           </div>
-          <div class="divider"></div>
-          <SvgIcon
-            name="show_all"
-            class="last showAllIcon"
-            @pointerdown="emit('fitToAllClicked')"
-          />
+        </div>
+        <div class="row clickableRow" @pointerdown.stop="emit('toggleCodeEditor')">
```

Comment:
It seems like we should `.stop` the pointerdown/pointerup/click events at the root element of the dropdown, and I think attaching these handlers to `click` would be more consistent with the way we're handling buttons now.

Reply:
Yeah those should be clicks. We should only use `pointerdown` in situations where we are dealing with drag interactions, so we need more control than what `click` provides.

Reply:
Replaced with click handlers